### PR TITLE
MCO: add etcd-quorum-guard image

### PR DESCRIPTION
**Pull request by runcom**
Needed to unblock https://github.com/openshift/installer/pull/1597

Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

**Comment by runcom** [comment]
Actually, re-following what we did for the InfraImage, this should go in with #613, merge that, then the installer PR is going to be merged, and then something like https://github.com/openshift/machine-config-operator/pull/518

omg, all of this is so silly

**Comment by RobertKrawitz** [comment]
So should I merge this into #613?

**Comment by runcom** [comment]
I believe so, and on top of this, you should also do something similar to this https://github.com/openshift/machine-config-operator/pull/508/files (I'm resyncing with Colin on this but this is what I would do following what we did with the infra image)

File: cmd/machine-config-operator/bootstrap.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ var (
 		setupEtcdEnvImage    string
 		infraImage           string
 		kubeClientAgentImage string
+		etcdQuorumGuardImage string
```
Comment (RobertKrawitz) [inline cmd/machine-config-operator/bootstrap.go]: Already in #613

File: cmd/machine-config-operator/bootstrap.go [hunk 2]
```diff
@@ -70,6 +71,8 @@ func init() {
 	bootstrapCmd.MarkFlagRequired("kube-client-agent-image")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.infraImage, "infra-image", "", "Image for Infra Containers.")
 	bootstrapCmd.MarkFlagRequired("infra-image")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdQuorumGuardImage, "etcd-quorum-guard-image", "registry.svc.ci.openshift.org/openshift/origin-v4.0:base", "Image for etcd Quorum Guard.")
```
Comment (RobertKrawitz) [inline cmd/machine-config-operator/bootstrap.go]: Already in #613 (including marking it required).

File: install/image-references [hunk 3]
```diff
@@ -27,6 +27,10 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-pod:v4.0
+  - name: etcd-quorum-guard-image
```
Comment (RobertKrawitz) [inline install/image-references]: Added just now to #613.

**Comment by RobertKrawitz** [comment]
@runcom is this even needed any longer, as all of the changes are in #613?

**Comment by runcom** [comment]
> @runcom is this even needed any longer, as all of the changes are in #613?

nope, this is closed since 4 days now, just tests rekicked for no good reason I guess

**Pull request closed without merging**